docker/clang-cuda/Dockerfile

```
@@ -1,13 +1,16 @@
 ARG TMP_DIR=/tmp
-FROM nvidia/cuda:10.1-devel-ubuntu18.04 AS tpl_toolchain_intersect_geosx_toolchain
+FROM nvidia/cuda:11.2.2-devel-ubuntu20.04 AS tpl_toolchain_intersect_geosx_toolchain
+
+ENV TZ=America/Los_Angeles
+RUN ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ > /etc/timezone
```

```suggestion
RUN ln -fs /usr/share/zoneinfo/America/Los_Angeles /etc/localtime && \
    dpkg-reconfigure -f noninteractive tzdata
```
If it works (it actually works on my images), I'd do this instead, to reduce the footprint (you're actually adding an `ENV` which we may not want).